packages/typeorm-class-validator-is-uniq/__tests__/index.ts

```diff
@@ -88,6 +90,29 @@ Array [
     });
   });
 
+  describe('handle nulls', () => {
+    const email = 'joe@example.com';
+    beforeAll(async () => {
+      const entities = [
+        repo.create({
+          email,
+          department: null,
+          company: 'JOIN Solutions AG',
+        }),
+        repo.create({
+          email: 'doe@example.com',
+          department: null,
+          company: 'ACME',
+        }),
+      ];
+      await repo.save(entities);
+    });
+    it('never consider null as unique value ', async () => {
+      const entity = await repo.findOneOrFail({ email });
```

Comment:
email here is not null

Reply:
email is not null but we need to check for validated values, it's a redundant query to match other tests.